packages/seclists/PKGBUILD

```diff
@@ -0,0 +1,25 @@
+# Maintainer heichblatt <git@hanneseichblatt.de>
+pkgname=seclists
+pkgver=r283.8ef8694
+pkgrel=1
+pkgdesc='A collection of multiple types of lists used during security assessments.'
+arch=('any')
+url='https://github.com/danielmiessler/SecLists/'
+license=('unknown')
+groups=('blackarch-exploitation')
+makedepends=('git')
+provides=("${pkgname}")
+source=('$pkgname::source_url')
```

Comment:
source_url is anything here. When I said source_url it's `https://github.com/danielmiessler/$pkgname`